supportedBackends/tes/src/test/scala/cromwell/backend/impl/tes/TesRuntimeAttributesSpec.scala

```diff
@@ -136,13 +136,17 @@ class TesRuntimeAttributesSpec extends WordSpecLike with Matchers {
       expectedDefaultsPlusUbuntuDocker
     )
   }
+  //TODO: RM add more tests for TES, same as JES?
```

Comment:
![](http://allthingsnext.com/wp-content/uploads/2016/04/yoda-do-or-do-not-there-is-no-try-star-wars-quote.jpg)